src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -36,6 +36,7 @@ public Priority(String specifiedPriority) {
      * Returns true if a given string is a valid priority.
      */
     public static boolean isValidPriority(String test) {
+        test = test.toUpperCase();
         return test.equals(VALIDATION_HIGH)
                 || test.equals(VALIDATION_NORMAL)
                 || test.equals(VALIDATION_LOW);
```

Comment:
Maybe we could reduce some lines of code here, since `VALIDATION_HIGH`, `VALIDATION_NORMAL` and `VALIDATION_LOW` are only used in this method.

```suggestion
        return Arrays.stream(PriorityName.values()).anyMatch(p -> test.equalsIgnoreCase(p.name()));
```

Reply:
Ok! It'll make the code more neat 👍